sympy/matrices/common.py

```diff
@@ -808,6 +808,7 @@ def diag(kls, *args, **kwargs):
         diagonal - to extract a diagonal
         .dense.diag
         .expressions.blockmatrix.BlockMatrix
+        .sparsetools.banded - to create multi-diagonal matrices
```

Comment:
@sylee957  do you have any idea why this is generating a Sphinx error saying that the source cannot be found?
```
Help on function banded in module sympy.matrices.sparsetools:

banded(*args, **kwargs)
```

Reply:
I think that the doc for banded should also be added to any rst files.
It may have lacked the physical link.